ci/scripts/deploy_master.sh

```diff
@@ -45,15 +43,29 @@ if [[ $GH_COMMIT_MESSAGE != *"ci(release): generate CHANGELOG.md for version"* &
     git commit -m "ci(build): release version ${GIT_TAG}"
     # Push commits and tags to origin branch
     git push --follow-tags origin $CIRCLE_BRANCH
+    # Create release with conventional-github-releaser
+    conventional-github-releaser -p angular -t $GITHUB_TOKEN
+    # Archive app
+    bundle exec fastlane archive
+    # Copy ipa file in artifacts folder
+    cp ${APPNAME}.ipa $CIRCLE_ARTIFACTS
+    # Upload ipa file to release
+    github-release upload \
+    --user $CIRCLE_PROJECT_USERNAME \
+    --repo $CIRCLE_PROJECT_REPONAME \
+    --tag ${GIT_TAG} \
+    --name "${APPNAME}.ipa" \
+    --file "$CIRCLE_ARTIFACTS/${APPNAME}.ipa"
 
     # Update CHANGELOG.md on gh-pages
     git branch -D gh-pages
     git fetch origin gh-pages
     git checkout gh-pages
     git checkout $CIRCLE_BRANCH CHANGELOG.md
 
-    # Create header content
-    HEADER="---\nlayout: modal\ntitle: changelog\n---\n"
+    # Create header content to CHANGELOG.md
+    HEADER="---"$'\r'"layout: modal"$'\r'"title: changelog"$'\r'"---"$'\r\r'
```

Comment:
Are you sure its works?

Reply:
yes, it was generated with this script
https://github.com/flyve-mdm/flyve-mdm-ios-inventory-agent/blob/gh-pages/CHANGELOG.md

Reply:
I think `\r\n` is most common
```
\r (Carriage Return)
\r\n (End Of Line) 
\n (Line Feed)
```

Reply:
I'll use `\r\n`

Reply:
changed